_50p_events/2018.md

```diff
@@ -6,13 +6,7 @@ datelocation: "8-9 February 2018, Bangalore"
 city: Bangalore
 start_time: 2018-02-08
 end_time: 2018-02-09
-description: "50p is India's first independently curated conference on payments. We bring stakeholders in the ecosystem to dialogue and collaborate on:
- - Building robust and secure engineering layers underlying payments;
- - Policies that affect the ecosystem as a whole; and
- - Scoping customer needs and developing payment products for the Indian market.
-
-If you are working on any of the above areas, you should come to 50p."
-boxoffice_item_collection: '685e9db7-b62d-4137-895f-f734dc216c4d'
+description: "50p is India's first independently curated conference on payments. The third edition of 50p recently concluded on 8-9 February 2018. For more events and announcements related to 50p, stay tuned."
```

Comment:
Removing information about previous edition does not help in any way. Boxoffice link can be removed.